Client/Frontend/Browser/URLBarView.swift

```diff
@@ -648,12 +648,21 @@ private class CurveView: UIView {
         return path
     }
 
+    private func leftCurvePath() -> UIBezierPath {
```

Comment:
Would it make sense to make this a lazy variable? Then we don't have to create the path every time we draw it?